feat: generate profile from tracee exec events

itaysk opened the pull request.

josedonizetti commented on action.yaml:
```diff
@@ -26,5 +21,11 @@ runs:
+  - shell: bash
+    run: |
+      echo "Generating profile..."
+      ./profile/profile-gen /tmp/tracee/out/trace_$GITHUB_RUN_ID.jsonl > /tmp/tracee/profile.json
```
The `> /tmp/tracee/profile.json` doesn't work, because once we map the volume in docker with `--privileged` this becomes a root folder, and the runner user doesn't have permission to create the file.

itaysk replied:
fixed it by writing the action's files into a different directory than tracee's. it might even make more sense this way

josedonizetti commented on profile/profile-gen:
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+jq -s -f ./exec.jq --slurpfile config exec-config.json "$1"
```
`exec-config.json` doesn't exist, so this command fails

itaysk replied:
leftover from my branch with control of which env goes into the pipeline.

itaysk committed:
feat: generate profile based on exec events

itaysk commented:
thanks for the review, I've tested this now and fixed some more bugs, and it works. note that I've removed args and env from this PR as they were too volatile, I will re-add them in a future PR with better control.

itaysk commented:
also added sorting to make the profile more stable

josedonizetti reviewed (approved):
Profile generated with this action https://github.com/josedonizetti/verified-build-tests/blob/main/tracee.profile
